src/main/java/org/broadinstitute/consent/http/service/VoteService.java

```diff
@@ -53,15 +53,26 @@ public VoteService(UserDAO userDAO, DatasetAssociationDAO dataSetAssociationDAO,
     }
 
     /**
-     * Find all votes for an election id.
+     * Find all votes for election ids.
      *
-     * @param electionIds The election ids for the election.
-     * @return Collection of votes for the given reference id
+     * @param electionIds The election ids for the elections.
+     * @return Collection of votes for the given reference ids
      */
     public List<Vote> findVotesByElectionIds(List<Integer> electionIds) {
         return voteDAO.findVotesByElectionIds(electionIds);
     }
 
+    /**
+     * Find all votes for an election id.
+     *
+     * @param electionId The election id for the election.
+     * @return Collection of votes on the election specified by the election id
+     */
+    public List<Vote> findVotesByElectionId(Integer electionId) {
+        return voteDAO.findVotesByElectionId(electionId);
```

Comment:
There are two cases to capture here:
* An existing election: we should do exactly what we're doing here and return the results, whatever they may be, empty list or not.
* No election for the id: we should throw a `NotFoundException`